github_repos.json

```diff
@@ -1,823 +1,723 @@
+        "ts": null
+      }
+    ]
+  },
+  {
+    "uuid": "fa181563-c7ea-4e5c-ac6c-0df919913bdd",
+    "sector": "Gov_Federal",
+    "ts": null,
+    "shortname": "swisscovid",
+    "name_de": "Swisscovid",
+    "orgs": [
+      {
+        "name": "SwissCovid",
+        "ts": null
+      }
+    ]
+  },
+  {
+    "uuid": "344bca77-9a0f-4764-814d-f7dc4eaf4e6d",
+    "sector": "Gov_Federal",
+    "ts": null,
+    "shortname": "sa",
+    "name_de": "Schweizer Armee",
+    "orgs": [
+      {
+        "name": "ZEM-Mediamatiker",
+        "ts": null
+      }
+    ]
+  },
+  {
+    "uuid": "5e2d8401-1122-4ccb-8798-10bb5e041fc0",
+    "sector": "Gov_Federal",
+    "ts": null,
+    "shortname": "visualize",
+    "name_de": "visualize.admin.ch",
+    "orgs": [
+      {
+        "name": "visualize-admin",
+        "ts": null
+      }
+    ]
+  },
+  {
+    "uuid": "d3d33379-1c3e-4039-9084-efd33371f26c",
+    "sector": "ResearchAndEducation",
+    "ts": null,
+    "shortname": "ubs",
+    "name_de": "Universität Basel",
+    "orgs": [
+      {
+        "name": "atmosphaerenwissenschaften",
+        "ts": null
+      }
+    ]
+  },
+  {
+    "uuid": "5c23efad-29af-4282-a868-78e2dd1ea33c",
+    "sector": "ResearchAndEducation",
+    "ts": null,
+    "shortname": "fhnw",
+    "name_de": "Fachhochschule Nordwestschweiz",
+    "orgs": [
+      {
+        "name": "FHNW",
+        "ts": null
+      },
+      {
+        "name": "fhnw-ise-qcrypt",
+        "ts": null
+      },
+      {
+        "name": "FHNW-Geomatik",
+        "ts": null
+      },
+      {
+        "name": "FHNW-IVGI",
+        "ts": null
+      },
+      {
+        "name": "fhnw-cs",
+        "ts": null
+      },
+      {
+        "name": "fhnw-biomedinf",
+        "ts": null
+      },
+      {
+        "name": "fhnw-bacnet-it",
+        "ts": null
+      },
+      {
+        "name": "fhgr2",
+        "ts": null
+      }
+    ]
+  },
+  {
+    "uuid": "10f6fe9f-1693-4ad0-9386-97ab7b9c783b",
+    "sector": "ResearchAndEducation",
+    "ts": null,
+    "shortname": "fhgr",
+    "name_de": "University of Applied Sciences of the Grisons",
+    "orgs": [
+      {
+        "name": "fhgr2",
+        "ts": null
+      }
+    ]
+  },
+  {
+    "uuid": "92a8c7a5-f7ac-4082-a823-a482b5d50c7f",
+    "sector": "ResearchAndEducation",
+    "ts": null,
+    "shortname": "epfl",
+    "name_de": "Universität EPFL",
+    "orgs": [
+      {
+        "name": "GameLab-UNIL-EPFL",
+        "ts": null
+      }
+    ]
+  },
+  {
+    "uuid": "a1d6e6fe-623d-448e-a7fd-f3625841c421",
+    "sector": "City",
+    "ts": null,
+    "shortname": "luzern",
+    "name_de": "Luzern",
+    "orgs": [
+      {
+        "name": "StadtLuzernIO",
+        "ts": null
+      }
+    ]
+  },
+  {
+    "uuid": "3d13b7b4-c21e-45a4-8b89-d82a5807a3b2",
+    "sector": "City",
+    "ts": null,
+    "shortname": "bern",
+    "name_de": "Bern",
+    "orgs": [
+      {
+        "name": "bkd-mba-fbi",
+        "ts": null
+      }
+    ]
+  },
```

Comment:
This probably belongs into the existing Stadt Bern:

https://github.com/digital-sustainability/oss-github-benchmark/blob/ebd8185fc0f170c0a8f696a5a979388b062aff08/github_repos.json#L3073-L3088

Also, is the sector "City" or "Gov_City"?